include/uv.h

```diff
@@ -243,6 +243,14 @@ typedef enum {
 UV_EXTERN unsigned int uv_version(void);
 UV_EXTERN const char* uv_version_string(void);
 
+typedef void * (*uv_malloc_cb)(size_t size);
+typedef void (*uv_free_cb)(void *ptr);
+
+UV_EXTERN void uv_replace_allocator(uv_malloc_cb malloc_cb, uv_free_cb free_cb);
+
+UV_EXTERN void * uv_malloc(size_t size);
```

Comment:
Style note: the libuv convention is `type* ptr` or `type* fun(void)`.  Can you update your PR accordingly?